sonar-plugin-api/src/main/java/org/sonar/api/rules/RuleAnnotationUtils.java

```diff
@@ -24,7 +24,9 @@
 
 /**
  * @since 2.3
+ * @deprecated since 5.6 sonar-check-api is deprecated and will be removed soon 
  */
+@Deprecated
 public final class RuleAnnotationUtils {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)